src/components/UI/GameList.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+import ContextProvider from '../../state/ContextProvider'
+import { GameStatus } from '../../types'
+import { getProgress } from '../../helper'
+const { ipcRenderer } = window.require('electron')
+interface List {
+    cover: string
+    logo: string
+    title: string
+    appName: string
+    isInstalled: boolean
+}
+
+interface InstallProgress {
+    percent: string
+    bytes: string
+    eta: string
+}
+
+const GameList = ({ cover, title, appName, isInstalled }: List) => {
+    const [progress, setProgress] = useState({
+        percent: '0.00%',
+        bytes: '0/0MB',
+        eta: '',
+    } as InstallProgress)
+    const { t } = useTranslation()
+
+    const { libraryStatus } = useContext(ContextProvider)
+
+    const gameStatus: GameStatus = libraryStatus.filter(
+        (game) => game.appName === appName
+    )[0]
+
+    const { status } = gameStatus || {}
+    const isInstalling = status === 'installing' || status === 'updating'
+    const isReparing = status === 'repairing'
+    const isMoving = status === 'moving'
+    const haveStatus = isMoving || isReparing || isInstalling
+
+    useEffect(() => {
+        const progressInterval = setInterval(async () => {
+            if (isInstalling) {
+                const progress = await ipcRenderer.invoke(
+                    'requestGameProgress',
+                    appName
+                )
+                setProgress(progress)
+            }
+        }, 1500)
+        return () => clearInterval(progressInterval)
+    }, [isInstalling, appName])
+
+    const { percent } = progress
+    const effectPercent = isInstalling
+        ? `${125 - getProgress(progress)}%`
+        : '100%'
+
+    function getStatus() {
+        if (isInstalling) {
+            return percent
+        }
+        if (isMoving) {
+            return t('gamelist.moving', 'Moving')
+        }
+        if (isReparing) {
+            return t('gamelist.repairing', 'Repairing')
+        }
+        return ''
+    }
+
+    return (
```

Comment:
I think maybe having a smaller component for Gamecard and another one for the GameList would be better. Then we could reuse all functions and change only the small component depending on the user selection.